crates/flipperzero/src/furi/string/pattern.rs

```diff
@@ -36,14 +36,17 @@ const FURI_STRING_FAILURE: usize = usize::MAX;
 pub trait Pattern: Sized {
     /// Checks whether the pattern matches anywhere in the haystack.
     #[inline]
+    #[allow(clippy::wrong_self_convention)] // same as in `std`
```

Comment:
If these are being moved onto the trait methods, then remove the blanket allow on the trait that I added in #83.

Reply:
Yup, thanks. Didn't see the top-level attribute when resolving the conflicts